e2e/playwright/testing-perspective-toggle.spec.ts

```diff
@@ -2,7 +2,11 @@ import { getUtils, orRunWhenFullSuiteEnabled } from '@e2e/playwright/test-utils'
 import { expect, test } from '@e2e/playwright/zoo-test'
 
 test.describe('Test toggling perspective', () => {
-  test('via command palette and toggle', async ({ page, homePage }) => {
+  test('via command palette and toggle', async ({
+    page,
+    homePage,
+    toolbar,
+  }) => {
     test.fixme(orRunWhenFullSuiteEnabled())
```

Comment:
This test is being skipped on PRs, so the changes you made to it won't have run on CI. You might consider removing this line if you've made the test more reliable and would like it to run.

Reply:
I'm down to try it! I was just concerned with removing all references to this toggle from the code base, so I saw this as making a future attempt to turn that test back on a little less confusing. I don't think that I've made the test more reliable but I can see.